src/xdp/trn_trace_prog_kern.h

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/**
+ * @file trn_trace_prog_kern.c
+ * @author ShixiongQi (@ShixiongQi)
+ *		   Sherif Abdelwahab (@zasherif)
```

Comment:
please remove my name. I did not author this code, and let's please make sure that the code is not copied from elsewhere and if so to have appropriate modifications, linking, or references. Thank you